pkg/features/kube_features.go

```diff
@@ -133,6 +133,12 @@ const (
 	// Taint nodes based on their condition status for 'NetworkUnavailable',
 	// 'MemoryPressure', 'OutOfDisk' and 'DiskPressure'.
 	TaintNodesByCondition utilfeature.Feature = "TaintNodesByCondition"
+
+	// owner: @sttts, @nikhita
+	// alpha: v1.8
+	//
+	// CustomResourceValidation is a list of validation methods for CustomResources
+	CustomResourceValidation utilfeature.Feature = "CustomResourceValidation"
```

Comment:
I think we can reference the one in `staging/src/k8s.io/apiextensions-apiserver/pkg/features/kube_features.go` here, can't we? Or even only below and drop this paragraph compeltely. Just follow `AdvancedAuditing`.